RELEASING.md

```diff
@@ -0,0 +1,49 @@
+# Release Process
+
+## Versioning
+The versioning follows the process described at [semver.org](https://semver.org/).
+
+### Public API
+We consider the public API of this project to be whatever is included in the
+`LIBRARY_PUBLIC_HEADERS` defined in `src/CMakeLists.txt`. Currently, this includes
+all the header files in `src/mococrw/`. Changes to the files will be versioned according
+to semantic versioning scheme.
+
+## PRs & Releasing
+
+### PR Checklist
+(probably to be extended)
+
+What should be done with every PR (if no new release is created):
+* CHANGELOG is updated with the details of the change (Unreleased section)
+
+### Releasing Checklist
+(probably to be extended)
+
+Check if the following steps have been performed **before** creating a new release. This includes
+all kinds of releases (Major, Minor and Patch releases)
+* README is updated if necessary
+* CHANGELOG is updated (add the new release)
+* Library .so version is increased (Major releases only)
```

Comment:
We should only increase the SOVERSION iff the ABI changes. There is no need to increase the SOVERSION if we do not break the ABI.

Also note that the library SOVERSION is not related to the MoCOCrW release version number. There is no need to have a MoCOCrW 2.0 install a `libmococorw.so.2` file.

Consequently, I'm not sure using 3.0 as initial version number is necessary.